.docksal/commands/release

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+## Make a new release and deploys it.
+##
+## Usage: fin release
+
+# Abort if anything fails
+set -e
+
+# Backup the environments.
+fin terminus backup:create ${hostingplatform}.dev --keep-for=365
+fin terminus backup:create ${hostingplatform}.test --keep-for=365
+fin terminus backup:create ${hostingplatform}.live --keep-for=365
+
+# Grab the live db.
+fin pull db -y --hosting-env=live
```

Comment:
This will give issues if there is no "live" environment.

Reply:
hmm, Can we check if an environment exists?  If so, we should fence test and live.

Reply:
Updated.